pkg/antctl/command_list.go

```diff
@@ -63,7 +63,7 @@ func (cl *commandList) applyToRootCommand(root *cobra.Command, client AntctlClie
 
 	for _, cmd := range cl.rawCommands {
 		if (runtime.Mode == runtime.ModeAgent && cmd.supportAgent) ||
-			(runtime.Mode == runtime.ModeController && cmd.supportController) {
+			(runtime.Mode == runtime.ModeController && cmd.supportController) || cmd.commandGroup == mc {
```

Comment:
I am trying to understand this. So the `mc` commands are *always* added, even when I run `antctl` from inside an antrea-agent Pod. Is this right?

Reply:
looks like it will be added no matter it's inside a Pod or not, @hjiajing  I think you need to check both cmd.supportController and cmd.supportAgent so it won't add `mc` sub-command when it's running in a Pod.

Reply:
done